eos/constraints/B-to-D-pi-l-nu.yaml

```diff
@@ -621,7 +621,7 @@ B^0->(Dpi)^+l^-nu::KinematicalDistribution[w]@Belle:2018A:
 # 2021
 #
 # {{{
-B^0->(Dpi)^+l^-nubar::KinematicDistributionsAngularObservables@BBGJvD:2021A:
+B^0->(Dpi)^+l^-nubar::KinematicDistributionsAngularObservables@BBGJvD:2020A:
```

Comment:
I think the paper is from 2021. Can you change the reference name instead of the constraint?

Reply:
Done